### PR TITLE
[Api Translate] Translate User and Role Section.

### PR DESCRIPTION
**Pull request by martineiber**
## Changes in this pull request
Part of #326

### CODE REVIEW

**Review by lukmzig — CHANGES_REQUESTED** [review]
Could you please re-base on this branch? https://github.com/pimcore/studio-backend-bundle/tree/326-api-translations

We also adapter all operation IDs for controllers so they match the translation key pre-fixes - therefore we should finish translations and notify Frontend before we merge the translation branch into the 1.x

**Comment by martineiber** [comment]
> Could you please re-base on this branch? https://github.com/pimcore/studio-backend-bundle/tree/326-api-translations
> 
> We also adapter all operation IDs for controllers so they match the translation key pre-fixes - therefore we should finish translations and notify Frontend before we merge the translation branch into the 1.x

Thx for the hint.... I will update this :)

**Comment by mattamon** [comment]
> > Could you please re-base on this branch? https://github.com/pimcore/studio-backend-bundle/tree/326-api-translations
> > We also adapter all operation IDs for controllers so they match the translation key pre-fixes - therefore we should finish translations and notify Frontend before we merge the translation branch into the 1.x
> 
> Thx for the hint.... I will update this :)

ah yeah and btw https://github.com/pimcore/studio-backend-bundle/pull/309#pullrequestreview-2232304735 :D

**Comment by mattamon** [comment]
Could you please resolve the conflict? :)

**Comment by martineiber** [comment]
> Could you please resolve the conflict? :)

Done

File: src/User/Controller/CreateUserController.php [hunk 1]
```diff
@@ -59,12 +59,12 @@ public function __construct(
     #[Post(
         path: self::API_PATH . '/user/',
         operationId: 'user_create',
-        summary: 'Create a new user.',
+        summary: 'create_user_summary',
```
Comment (lukmzig) [inline src/User/Controller/CreateUserController.php]: can you please change the translation keys so they match the operationIDs? 
e.g here `user_create_summary`

the idea would be that all of elements (assets, data objects, users ....) are then together when we sort the translations

**Review by mattamon — APPROVED** [review]
Just as a hint we also added more detailed descriptions to the routes, based in the parameters, filters etc. 

e.g. 

```yaml
property_update_description: |
  Updating the property with the given <strong>{id}</strong>. <br>
  See the full description of updatable fields with the schema <strong>UpdatePredefinedProperty</strong>
```

File: src/User/Controller/CreateUserController.php (hunk 1 above)
Comment (martineiber) [reply src/User/Controller/CreateUserController.php]: thx... missed that one after the merge :)

Done.

**Review by lukmzig — APPROVED** [review]
:turtle: :rainbow:

**Pull request merged**